### DISCUSSION TITLE
Save record table as pretty table

### DISCUSSION THREAD

**Discussion by nicholaschiasson** [nushell/nushell#5741] [Q&A] [answered]
Hi, I wanted to ask if there was a way to `save` a table to a file with the same display format that is output to the nushell standard output.

For example if I have some dummy data like this

```
〉[{field1: "value", field2: 5}]                                                                                                06/08/2022 01:28:34 PM
╭───┬────────┬────────╮
│ # │ field1 │ field2 │
├───┼────────┼────────┤
│ 0 │ value  │      5 │
╰───┴────────┴────────╯
```

Doing something like `〉[{field1: "value", field2: 5}] | to text | save test.txt` looks like it just stringifies the nuon, writing `[{field1: value, field2: 5}]` to the file. I think it's good behaviour, but is there a way I can have the nice table stringified, just for prettier output to files if I wanted?

**Comment by fdncred** [comment]
On WSL i can do `ls | vim -` and it loads in vim where i could save it. I think that will work with code too like `ls | code -`

**Reply by nicholaschiasson** [reply]
Nice tip! On windows though it looks pretty ugly because the characters used for borders seem to be some unicode that doesn't get processed well... or something 😕

**Reply by fdncred** [reply]
you're right. it looks terrible on windows. i wonder why that is?

**Reply by fdncred** [reply]
I figured it out I think on Windows with vim. I added this to my `_vimrc`
```
set encoding=utf-8
set termencoding=utf-8
```
once i did that on Windows I could do `ls | vim -` and it worked fine, however `code` still looks messed up even though the encoding is utf-8 so I'm not sure what the deal with `code` is.

**Comment by panicbit** [comment, marked as answer]
How about:
`ls | table | ansi strip | save output.txt`

**Reply by panicbit** [reply]
`table` renders the table to a string and `ansi strip` removes all the colors / ansi escape codes from it.

**Reply by fdncred** [reply]
oh ya, good idea. I did that the other day. It's so hard to remember all the things one can do with nushell. LOL

**Reply by nicholaschiasson** [reply]
Nice! I think this is probably actually the best answer.

**Comment by DejayRezme** [comment]
Not the same format, but `ls | to md` converts to markdown
